cmd/kube-scheduler/app/options/insecure_serving.go

```diff
@@ -153,7 +153,7 @@ func (o *CombinedInsecureServingOptions) Validate() []error {
 	errors := []error{}
 
 	if o.BindPort <= 0 || o.BindPort > 32767 {
-		errors = append(errors, fmt.Errorf("--insecure-port %v must be between 0 and 32767, inclusive. 0 for turning off insecure (HTTP) port", o.BindPort))
```

Comment:
this reminds me: can we add --insecure-port and deprecate --port?

Reply:
If we want to deprecate insecure http, do we have the need to do this?

Reply:
in fact no